### PR TITLE
Modify detection notebook and save pytorch model

### LINKED ISSUE #176: Refactor detection notebook
- [x] current object detection notebook should be modified to enable auto conversion of mmdet model.
- [x] pytorch native model should be saved and registered

### PR DESCRIPTION
**Pull request by univerone**
Closes #176

since hummingbird has removes ONNX_INPUTS in current pr https://github.com/microsoft/hummingbird/pull/378,
we should remove this to avoid error.

#### Does this change require updates to documentation?

No
#### Before submitting this PR, please make sure you have:

- [x] run `python -m pytest tests/` and confirmed all checks still pass.
- [x] verified that any code or assets from external sources are properly credited.

### CODE REVIEW

File: modelci/hub/converter.py [hunk 1]
```diff
@@ -123,6 +122,20 @@ def from_onnx(
 
 __all__ = ['TorchScriptConverter', 'TFSConverter', 'ONNXConverter', 'TRTConverter', 'to_tvm']
 
+class PYTConverter(object):
```
Comment (YuanmingLeee) [inline modelci/hub/converter.py:125]: it should be named as `PyTorchExporter` and put somewhere else.

File: modelci/types/bo/model_objects.py [hunk 2]
```diff
@@ -43,7 +43,8 @@ class Engine(Enum):
     ONNX = 3
     TRT = 4
     TVM = 5
-    CUSTOMIZED = 6
+    PYT = 6
```
Comment (YuanmingLeee) [inline modelci/types/bo/model_objects.py:46]: Change to PYTORCH

**Review by YuanmingLeee — CHANGES_REQUESTED** [review]

File: modelci/hub/converter.py (hunk 1 above)
Comment (univerone) [reply modelci/hub/converter.py:125]: > it should be named as `PyTorchExporter` and put somewhere else.
How about just put it into register process?

File: modelci/types/bo/model_objects.py (hunk 2 above)
Comment (univerone) [reply modelci/types/bo/model_objects.py:46]: OK!

File: modelci/hub/converter.py (hunk 1 above)
Comment (YuanmingLeee) [reply modelci/hub/converter.py:125]: good

**Comment by huangyz0918** [comment]
seems like your title/subtitle didn't got formatted @univerone .

Screenshot 2021-01-14 at 12 02 11 PM

**Comment by univerone** [comment]
> seems like your title/subtitle didn't get formatted @univerone .
> 
> Screenshot 2021-01-14 at 12 02 11 PM

I'll resolve that, there exists unnecessary indent

File: modelci/hub/manager.py [hunk 3]
```diff
@@ -139,7 +140,7 @@ def register_model(
             version=version)[0]
 
         # profile registered model
-        if profile:
+        if profile and engine != Engine.PYT:
```
Comment (YuanmingLeee) [inline modelci/hub/manager.py:143]: `engine != Engine.PYTORCH`

Comment (univerone) [reply modelci/hub/manager.py]: I'll fix it!

**Review by YuanmingLeee — APPROVED** [review]

**Pull request merged**